sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryContainOperator.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// <see href="https://docs.microsoft.com/en-us/azure/digital-twins/reference-query-operators#contains-operators">Contains operators</see> in the ADT Query language.
+    /// </summary>
+    public enum QueryContainOperator
+    {
+        /// <summary>
+        /// IN operator defined by the ADT query language.
+        /// </summary>
+        IN = 1,
```

Comment:
`In` and `NotIn` you can map the `NotIn` to NIN internally when you do the lookup

Reply:
Yep, should not be all caps. We match the casing style of the language we're in.